diff --git a/node_modules/@react-navigation/drawer/src/views/modern/Overlay.tsx b/node_modules/@react-navigation/drawer/src/views/modern/Overlay.tsx
index c91d08d..9acdcc6 100644
--- a/node_modules/@react-navigation/drawer/src/views/modern/Overlay.tsx
+++ b/node_modules/@react-navigation/drawer/src/views/modern/Overlay.tsx
@@ -42,7 +42,7 @@ const Overlay = React.forwardRef(function Overlay(
       style={[styles.overlay, overlayStyle, animatedStyle, style]}
       animatedProps={animatedProps}
     >
-      <Pressable onPress={onPress} style={styles.pressable} />
+      <Pressable onPress={onPress} style={styles.pressable} pointerEvents="none" />
     </Animated.View>
   );
 });
@@ -63,7 +63,6 @@ const styles = StyleSheet.create({
   },
   pressable: {
     flex: 1,
-    pointerEvents: 'auto',
   },
 });
 
